pyvista/plotting/qt_plotting.py

```diff
@@ -845,14 +852,14 @@ def add_callback(self, func, interval=1000, count=None):
             `func` will be called until the main window is closed.
 
         """
-        timer = QTimer(parent=self.app_window)
-        timer.timeout.connect(func)
-        timer.start(interval)
-        self.app_window.signal_close.connect(timer.stop)
+        self.callback_timer = QTimer(parent=self.app_window)
```

Comment:
This member is only used for testing, isn't it? If so I'd prepend it with an underscore to explicitly make it not mark it as not part of the exposed class interface.

Reply:
I realize it would be so much better to just return the timer actually. Maybe in another PR.